dermaknight/app/ui/quizComponents/QuizPage.module.css

```diff
@@ -0,0 +1,82 @@
+.buttonsContainer {
+    position: fixed;
+    top: 720px;
+    left: 0;
+    right: 0;
+    display: flex;
+    justify-content: center;
+    padding: 20px;
+
+
+    z-index: 1000;
+}
+.progressBarandExitContainer{
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+
+}
+.exitButton{
+    border:0;
+    background-color: transparent;
+    
+}
+
+.previousButton {
+
+    height: 49px;
```

Comment:
Avoid setting explicit heights on things. Instead use padding to add height.
If you need to make them the same height, put inside a flex container and then use `align-item: stretch`